var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -227,13 +227,11 @@ def cmake_args(self):
         return options
 
     # Delegate in the vtk-m built smoke test
-    def smoke_test(self):
+    def test_smoke_test(self):
```

Comment:
This should not start with `test_` or it will be picked up as a separate stand-alone test from `test_build`.

Reply:
```
======================= SUMMARY: vtk-m-2.1.0-rc2-3yf3jfg =======================
VtkM::test_build .. PASSED
VtkM::test_smoke_test .. PASSED
============================= 2 passed of 2 parts ============================
```

Reply:
Or just drop `test_build` as we discussed offline.